src/engine/bank20.asm

```diff
@@ -533,17 +684,730 @@ MapDataPointers: ; 80e5d (20:4e5d)
 	dw MapDataPointers_81697
 
 MapDataPointers_80e67: ; 80e67 (20:4e67)
-	INCROM $80e67, $8100f
+	db $1b, $59, $00, $00
```

Comment:
The first two bytes look like a pointer. Do we know what the last two bytes are for?

Reply:
The third byte is definitely the bank number (relative to bank 20), the last one I have no idea, I don't believe a function to use these values has been disasm'd yet. (The only way to get to a pointer in this table is to call GetMapDataPointer with either 0, which doesn't happen in the current code that's been uncovered. Although Func_803b9 calls GetMapDataPointer  with a value in WRAM, wd131.)